### DISCUSSION TITLE
Bootstrap node for libp2p-workshop?

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#4154] [Q&A] [answered]
I am playing around with the [libp2p-workshop](https://github.com/thomaseizinger/libp2p-workshop.git) project that @thomaseizinger presented at IPFSþing 2023 and I noticed that to make the final iteration work, the app must connect to a bootstrap node. I have a couple of questions:

1. Is there a repo that has the code for the bootstrap node referenced in the README.md file in the `iteration-final` branch?
2. What would it take to add code to the app so that it can function as a bootstrap node instead of a client?
3. What code is necessary to have bootstrap functionality in an app?

Cheers! 🍻

**Comment by dhuseby** [comment, marked as answer]
Nvrmnd...I figured it all out.

**Reply by thomaseizinger** [reply]
Do you mind posting a summary? :)

There is no particular bootstrap node necessary, just _another_ node that speaks the same protocols. But can I check whether I still have the code somewhere.
